pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -2446,9 +2446,10 @@ class HdRprApiImpl {
         }
 
         // Though if adaptive sampling is enabled in a batch session we first render m_minSamples samples
-        // and after that render 1 sample at a time because we want to query the current amount of
+        // and after that, if needed, render 1 sample at a time because we want to check the current amount of
         // active pixels as often as possible
         const bool isAdaptiveSamplingEnabled = IsAdaptiveSamplingEnabled();
+        const bool isActivePixelCountCheckRequired = isAdaptiveSamplingEnabled && m_rprContextMetadata.pluginType == kPluginTahoe;
```

Comment:
Adaptive sampling works for Northstar too.  Or do you just not check ever sample if Northstar here?

Reply:
We need to do a manual check only for Tahoe, that's what this flag is for.